tilequeue/wof.py

```diff
@@ -330,10 +332,25 @@ def failure(reason):
     cessation = min(edtf_cessation.latest_date(),
                     edtf_deprecated.latest_date())
 
+    # grab any names in other languages
+    lang_suffix_size = len('_preferred') * -1
+    l10n_names = {}
+    for k, v in props.iteritems():
+        if not v:
+            continue
+        if not k.startswith('name:') or not k.endswith('_preferred'):
+            continue
+        if isinstance(v, list):
+            v = v[0]
+        lang = k[:lang_suffix_size]
```

Comment:
Yea, we'll get something that looks like 3 letter code_ wof name type. For example: `name:eng_x`

Would we ever care about the name type? What should we store on our end, just `name:eng`?

As it stands now, I'm taking only the preferred names, and just the first one at that. I'm expecting that in the tile we'll only show a single value for a language code, to match how it works with the osm tags. Should we take a different approach to select the best candidate?